src/harness/sas_interface.py

```diff
@@ -447,6 +447,25 @@ def TriggerDpaDeactivation(self, request):
     """   
     pass
 
+  @abc.abstractmethod
+  def QueryPropagationAndAntennaModel(self, request):
+    """SAS admin interface to query propagation and antenna gains for CBSD and FSS  or Provided PPA Contour
+    
+    Args:
+      request: A dictionary with multiple key-value pairs where the keys are
+        reliabilityLevel: (permitted values: -1, 0.05, 0.95) 
+        cbsd: dictionaty defining cbsd
+        fss(optional): dictionaty defining fss
+        ppa(optional): GeoJSON Object
```

Comment:
I think that technically `ppa` is not itself a GeoJSON object. Instead it has a field `contour` which is of type GeoJSON.